src/main_measurements.cpp

```diff
@@ -27,7 +27,11 @@ static void post_sentinel(int signum)
 {
   (void)signum;
   const char msg[] = "\nGot signal - shutting down\n";
-  write(STDERR_FILENO, msg, sizeof(msg));
+  ssize_t ret = write(STDERR_FILENO, msg, sizeof(msg));
+  if (ret == -1) {
+    std::cerr << "write() failed" << std::endl;
```

Comment:
I'm not sure that this is reentrant, so I don't think it should be used in a signal handler. That's why we're using `write` to begin with.

I think we really do want to ignore the return value somehow - there really isn't anything we can do if the write fails.